README.md

```diff
@@ -1,24 +1,64 @@
-# README
+# Choreganize
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is the "frontend" repository for the Chorganize application. It is responsible for authenticating the user and making api requests to our [backend application](https://github.com/choreorganize/choreganize-api).
 
-Things you may want to cover:
+### About this Project
 
-* Ruby version
+Chorganize is an application in which users can create, complete, and monitor chores for their household. Each chore is weighted by difficulty to ensure fairness across all the users of a household.
 
-* System dependencies
+### Database Schema
+<!-- upload image here -->
 
-* Configuration
+### Getting Started
 
-* Database creation
+These instructions will give you a copy of the project up and running on
+your local machine for development and testing purposes.
 
-* Database initialization
+## Local Setup
 
-* How to run the test suite
+1. Fork and Clone the repo
+2. Install gem packages: `bundle install`
+3. Setup the database: `rails db:create`
+4. Run migrations: ` rails db:migrate`
+5. Install the Figaro gem: `bundle exec figaro install`
+6. Create an account with [Open Weather Map](https://home.openweathermap.org/users/sign_up) and request an API key
```

Comment:
I don't know if the front end will need the weather key. we may need to remove this is it turns out it does not.

Reply:
I was thinking the same thing! It might be backwards but I've always created the README at the end of the project so I know what I'm talking about haha.